pontoon/projects/forms.py

```diff
@@ -1,7 +1,18 @@
+import bleach
+
 from django import forms
 from django.core import validators
+from django.conf import settings
 
 
 class NotificationsForm(forms.Form):
     message = forms.CharField(widget=forms.Textarea)
     selected_locales = forms.CharField(validators=[validators.validate_comma_separated_integer_list])
+
+    def clean_message(self):
+        message = self.cleaned_data['message']
```

Comment:
I'd move that code in the validator/django form field class and add tests for views with input sanitization.